WBooks/BookLibrary/BooksTableView.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  BookTableView.swift
+//  WBooks
+//
+//  Created by Federico Agustin Diaz on 25/07/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class BooksTableView: UITableViewCell, NibLoadable {
+    
+    @IBOutlet weak var table: UITableView! {
+    didSet {
```

Comment:
I don't know if is github, but check this indentation please

Reply:
Yup it was my problem i already fixed it